frontpage-api/src/main/scala/no/ndla/frontpageapi/model/api/FrontPage.scala

```diff
@@ -19,7 +19,8 @@ import scala.annotation.unused
 @description("The Menu object")
 case class Menu(
     @description("Id of the article") articleId: Long,
-    @description("List of submenu objects") menu: List[MenuData]
+    @description("List of submenu objects") menu: List[MenuData],
+    @description("Hide submenu") hideLevel: Boolean
```

Comment:
Ed-redigering må oppdateres for at det skal være mogeleg å sette denne verdien. Kan det være en ide å legge inn en default-verdi en plass dersom feltet ikkje er satt, slik at vi kan deploye apiet og så utvide ed i etterkant?
```suggestion
    @description("Hide this level in menu") hideLevel: Boolean
```